.github/workflows/update-intellij-version.yml

```diff
@@ -0,0 +1,34 @@
+name: Update IntelliJ Version
+
+on:
+  schedule:
+    - cron: '0 1 * * 1'
+
+jobs:
+  bump-intellij-version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Update default IntelliJ version
+        run: |
+          sbt "update-intellij / run --java-home /usr/local/openjdk-11"
```

Comment:
What `java-home` is assumed by default? 🤔

Reply:
For now the default JDK version on `ubuntu-latest` is:
`openjdk version "11.0.16.1" 2022-08-12`
but I assume that it could change in time.